src/main/java/frc/robot/subsystems/Wrist.java

```diff
@@ -129,35 +152,59 @@ public void setEnabled(boolean enable) {
    * @return the absolute encoder position in rotations, zero being facing forward
    */
   public double getAbsEncoderPos() {
-    if (RobotBase.isSimulation()) return m_armSim.getAngleRads();
+    // This is the wrong way to simulate the encoder. Should just use m_absEncoder.getDistance()
+    // if (RobotBase.isSimulation()) return m_armSim.getAngleRads();
     // inverted to make rotating towards stow positive
     // offset makes flat, facing out, zero
-    return (-m_absEncoder.getAbsolutePosition() + WristConstants.kEncoderOffset) * 2 * Math.PI; 
+    // return (-m_absEncoder.getAbsolutePosition() + WristConstants.kEncoderOffset) * 2 * Math.PI; 
```

Comment:
the reason for that was that we thought the encoderSim was broken. We can go over it with Michael C. who spent a while trying to get it to work and eventually just did that

Reply:
DutyCycleEncoderSim.setDistance() is broken. There's a bug report and a fix. It should be fixed in the next release. A workaround is used.